x-pack/plugins/security/public/analytics/register_user_context.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { catchError, from, map, of } from 'rxjs';
+
+import type { AnalyticsServiceSetup } from '@kbn/core/public';
+import { Sha256 } from '@kbn/crypto-browser';
+
+import type { AuthenticationServiceSetup } from '..';
+
+/**
+ * Set up the Analytics context provider for the User information.
+ * @param analytics Core's Analytics service. The Setup contract.
+ * @param authc {@link AuthenticationServiceSetup} used to get the current user's information
+ * @param cloudId The Cloud Org ID.
+ * @private
+ */
+export function registerUserContext(
+  analytics: AnalyticsServiceSetup,
+  authc: AuthenticationServiceSetup,
+  cloudId?: string
+) {
+  analytics.registerContextProvider({
+    name: 'user_id',
```

Comment:
When this lived in the Cloud plugin, it was called `cloud_user_id`. Is the name change intentional?